generators/typescript-axios/templates/README.mustache

```diff
@@ -0,0 +1,143 @@
+# Onfido Node.js Library
+
+The official Node.js library for integrating with the Onfido API.
+
+Documentation can be found at <https://documentation.onfido.com>
+
+This library is only for use on the backend, as it uses Onfido API tokens which must be kept secret. If you do need to collect applicant data in the frontend of your application, we recommend that you use the Onfido SDKs: [iOS](https://github.com/onfido/onfido-ios-sdk), [Android](https://github.com/onfido/onfido-android-sdk), [Web](https://github.com/onfido/onfido-sdk-ui), and [React Native](https://github.com/onfido/react-native-sdk).
+
+This version uses Onfido API {{ apiVersion }}. Refer to our [API versioning guide](https://developers.onfido.com/guide/api-versioning-policy#client-libraries) for details of which client library versions use which versions of the API.
+
+## Installation
+
+For npm:
+
+```sh
+npm install @onfido/api
+```
+
+For Yarn:
+
+```sh
+yarn add @onfido/api
+```
+
+## Getting started
+
+Require the package:
+
+```js
+const { DefaultApi, Configuration, WebhookEventVerifier } = require("@onfido/api");
+```
+
+For TypeScript users, types are available as well:
+
+```ts
+import { DefaultApi, Configuration, WebhookEventVerifier } from "@onfido/api";
+import { isAxiosError } from "axios";
+
+```
+
+Configure with your API token and region:
+
+```js
+const onfido = new DefaultApi({
+  apiToken: process.env.ONFIDO_API_TOKEN,
+  // Supports Region.EU, Region.US and Region.CA
+  region: Region.EU
+});
+```
+
+Using with `async`/`await` (in an `async function`):
+
+```js
+try {
+  const applicant = await onfido.createApplicant({
+    first_name: "Jane",
+    last_name: "Doe",
+    location: {
+      ip_address: "127.0.0.1",
+      country_of_residence: "GBR"
+    }
+  });
+
+  const check = await onfido.createCheck({
+    applicantId: applicant.data.id,
+    reportNames: ["identity_enhanced"]
+  });
+
+  return check;
+} catch (error) {
+  const error_details = error.response?.data.error;
+
+  if (isAxiosError(error) && error_details)  {
+    // An error response was received from the Onfido API, extra info is available.
+    console.log(error_details.message);
+    console.log(error_details.type);
```

Comment:
What is type in here? Let's give an example of the status code and a link to the axios documentation given there is no data structure to represent the error